### ISSUE TITLE
500 internal server error when trying to sign up

### ISSUE THREAD

**Issue by theharrisonmoore** [yalla-coop/earwig#845]
@thejoefriel @dupreesi @RamyAlshurafa @amusameh 500 internal server error when trying to sign up both as a non-worker _and_ as a worker.
https://drive.google.com/open?id=13pGPO33JNt9S2Q8vvOAndygfhaVFQ8-j

**Comment by amusameh** [comment]
Hi @theharrisonmoore , Have you changed anything in your mail-chimp account?

**Comment by theharrisonmoore** [comment]
Oh dear! I completely forgot to tell you about it! Stan is helping me test
Mailchimp and make sure it's authenticated properly so that emails are
being delivered successfully. To do this, we've removed (and saved) the
list of contacts and added a new list of test emails from GlockApps (an
email-testing programme). Once Stan and I have finished the testing, I'll
remove the list and replace it with the original contacts.

In terms of actually changing anything, no we haven't. We've only
authenticated both domains (earwig.work and earwigwork.com) by adding CNAME
and TXT records to Wix

If you go to Mailchimp, you'll see 78 contacts. These are the temporary
test emails.

Best,
Harrison Moore
Sparky & founder
earwig


On Fri, 29 Nov 2019 at 11:11, Abdalsamad Y. Abumusameh <
notifications@github.com> wrote:
>
> Hi @theharrisonmoore , Have you changed anything in your mail-chimp
account?
